glances_api/__init__.py

```diff
@@ -33,13 +34,16 @@ def __init__(
         self.username = username
         self.password = password
         self.verify_ssl = verify_ssl
+        self.httpx_client = httpx_client
 
     async def get_data(self, endpoint):
         """Retrieve the data."""
         url = "{}/{}".format(self.url, endpoint)
 
+        httpx_client = self.httpx_client if self.httpx_client else httpx.AsyncClient(verify=self.verify_ssl)
```

Comment:
@fabaff - note that you're creating a new instance of httpx Async Client on each call, is this intentional?

Reply:
I can't remember why I did it that way. Could be that i simple used the same approach for httpx that was there before.

Reply:
I'm not  sure if this should be reinitiated each time or can be reused.

I believe that the only problem in reusing the client one time is if there's an exception in the method or somehow the client is closed, the `httpx_client` won't be reinitialized (as far as I understand python behavior from being a Java developer 😕 ).